LogMonitor/LogMonitorTests/LogMonitorTests.cpp

```diff
@@ -9,7 +9,7 @@
 #include "../src/LogMonitor/EtwMonitor.cpp"
 #include "../src/LogMonitor/EventMonitor.cpp"
 #include "../src/LogMonitor/JsonFileParser.cpp"
-#include "../src/LogMonitor/FileMonitor/Utilities.cpp"
+#include "../src/LogMonitor/FileMonitor/FileMonitorUtilities.cpp"

```

Comment:
Why is this utility file specific to files? can the functions here be moved in LogFileMonitor.cpp (if they are specific file monitoring) or the Utility.cpp file (if they can be utilized throughout the tool)?

Reply:
The methods are only specific to LogFileMonitor. Moving them to LogFileMonitor.cpp will make the file too long a difficult to manage. Hence the need to seperate the utilities methods